webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -40,31 +33,37 @@ export default function EditUserForm({
   const [status, setStatus] = useState<EditUserFormSubmitStatus>(
     initialSubmitFormStatus,
   );
```

Comment:
```suggestion
```

suggestion (if-minor): We could remove this `useState`, the `useEffect`, and the types associated with it because we don't use them anymore.